superset-frontend/src/cccs-viz/plugins/plugin-chart-iframe/src/IFrameVisualization.tsx

```diff
@@ -1,27 +1,52 @@
-import React from 'react';
+import { css, SupersetTheme } from '@superset-ui/core';
+import React, { useMemo } from 'react';
 import { IFrameVisualizationProps } from './types';
 
+const errorStyles = (theme: SupersetTheme) => css`
+  color: ${theme.colors.error.base};
+  font-weight: bold;
+`;
+
 export default function IFrameVisualization(props: IFrameVisualizationProps) {
   const {
+    errorMessage,
     url,
     url_parameter_value,
     parameter_name,
     parameter_prefix,
-    errorMessage,
   } = props;
 
   // We encode the parameter value (including its prefix)
   // so the Fission proxy doesn't break on parameters that contain, for example, URLs.
-  const encodedUrl = `${url}?${parameter_name}=${
-    parameter_prefix ? encodeURIComponent(parameter_prefix) : ''
-  }${encodeURIComponent(url_parameter_value)}`;
+  const encodedUrl = useMemo(
+    () =>
+      `${url}?${parameter_name}=${
+        parameter_prefix ? encodeURIComponent(parameter_prefix) : ''
+      }${encodeURIComponent(url_parameter_value)}`,
+    [parameter_name, parameter_prefix, url, url_parameter_value],
+  );
 
   return (
     <>
       {errorMessage ? (
-        <>{errorMessage}</>
+        errorMessage !== 'fission-incompat' ? (
+          <span css={errorStyles}>{errorMessage}</span>
+        ) : (
+          <span css={errorStyles}>
+            Fission endpoints are not supported in{' '}
+            {navigator.userAgent.includes('Edg/')
+              ? 'Microsoft Edge'
+              : 'Google Chrome'}
```

Comment:
Did we make a ticket to re-support Chrome? (If I'm remembering right and it should work there.) Dropping this comment here because I should be gone already and I'm not going to remember to check the backlog next week if I don't make this comment.

Reply:
https://cccs.atlassian.net/browse/CLDN-2429